cookbooks/travis_internal_base/files/default/00-create-users

```diff
@@ -61,8 +61,10 @@ __add_github_keys() {
   mkdir -p "${ssh_dir}"
   chmod 0700 "${ssh_dir}"
 
-  ssh-import-id -o "${ssh_dir}/authorized_keys" "gh:${github_username}"
+  export URL=https://github.com/%s.keys
+  ssh-import-id -o "${ssh_dir}/authorized_keys" "${github_username}"
   chown -R "${username}:${username}" "${ssh_dir}"
+  unset URL
```

Comment:
It's a nitpick, but I'd rather see this as a one-liner a la `URL=https://... ssh-import-id -o ...` without the leading `export` and the trailing `unset` 😁